QUICKSTART.md

```diff
@@ -48,6 +47,10 @@ git remote add heroku git@heroku.com:appinfive.git
 vim Gemfile
   add
     gem 'shopify_app'
+    
+or simply run the following command 
+
+$ echo "gem 'shopify_app', '~> 7.0.0'" >> Gemfile
```

Comment:
I've just updated this command!

Best,
Gopi

Code Rapper - Shopify Experts
<http://t.sidekickopen44.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs2BFWq2W3MhxYF1q7s-KW64k9XR56dPlnf8hWvLT02?t=https%3A%2F%2Fexperts.shopify.com%2Fcode-rapper%2F%3Fref%3Dcode-rapper&si=5239261854957568&pi=7b19e084-fd56-45c7-ceaf-56704de143ea>
Skype: gopinath.para | +91 8050848189

Availability : 2pm - 10pm (IST) / 4.30am - 12.30pm (EDT) / 8.30am - 4.30pm
(GMT)

On Tue, Mar 15, 2016 at 8:57 PM, Kevin Hughes <notifications@github.com>
wrote:

> In QUICKSTART.md
> <https://github.com/Shopify/shopify_app/pull/234#discussion_r56182553>:
>
> > @@ -48,6 +47,10 @@ git remote add heroku git@heroku.com:appinfive.git
> >  vim Gemfile
> >    add
> >      gem 'shopify_app'
> > +
> > +or simply run the following command
> > +
> > +$ echo "gem 'shopify_app', '~> 7.0.0'" >> Gemfile
>
> Lets just replace the existing instruction with this (remove the vim
> stuff). Can you remove the version number from the echo?
>
> I added a note on master about using the latest version so we're covered
> there now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/Shopify/shopify_app/pull/234/files#r56182553
>